src/api/cart.js

```diff
@@ -2,23 +2,57 @@ import Base from './base';
 import Hooks from '../hooks';
 
 export default class extends Base {
+    /**
+     * Get a collection of Carts. For now, this will only return an array of a single cart as multiple carts per session
+     * are not currently supported.
+     *
+     * @param options
+     * @param {Function} callback
+     */
+    getCarts(options = {}, callback) {
+        let url = '/api/storefront/carts';
+
+        if (options.includeOptions) {
+            url = this.includeOptions(url);
+        }
+
+        this.makeRequest(url, 'GET', options, true, (err, response) => {
+            callback(err, response);
+        });
+    }
+
     /**
      * Get the current Cart's details, either with or without Product Option selections.
+     * Can also be used to get a particular cart provided a cartId in the options.
      *
      * @param options
      * @param {Function} callback
      */
     getCart(options = {}, callback) {
-        let url = '/api/storefront/cart';
+        /* If no cart ID is provided, get the collection of carts and return the first one */
+        if (!options.cartId) {
+            return this.getCarts(options, (err, response) => callback(err, response[0]));
+        }
+
+        let url = `/api/storefront/carts/${options.cartId}`;
 
         if (options.includeOptions) {
-            url = `${url}?include=lineItems.physicalItems.options,lineItems.digitalItems.options`;
+            url = this.includeOptions(url);
         }
+
         this.makeRequest(url, 'GET', options, true, (err, response) => {
             callback(err, response);
         });
     }
 
+    /**
+     * Add the parameters to a URL needed to get product option details on cart line items
+     * @param url
+     */
+    includeOptions(url) {
+        return `${url}?include=lineItems.physicalItems.options,lineItems.digitalItems.options`;
```

Comment:
if we are making a breaking change I would prefer to break this options as well to be included based on user input. Thoughts ??

Reply:
It is included based on user input to the getCart method (params)

Reply:
No I mean user input options like `includePhysicalItems = true`, `includeDigitalItems = true` and we just include the ones requested.

Reply:
Gotcha, you think it should be granular. I'll make the update.

Reply:
Ya if a store does not sell digital products why should we make the call and force the backend to even check.

Reply:
Ah, so to clarify - the API will always return digital items and physical items. These flags only control if product option selections are returned on those items.

I combined them because I didn't think the differentiation added a lot of value - you probably want to just get options on all items or on none of them.

For example, in Cornerstone, since we need to be agnostic of whatever might be in the catalog, we'll always request options for both types anywhere we need option details.